tools/appsre-build-fedora-worker-packer.sh

```diff
@@ -8,4 +8,20 @@ export BUILD_RPMS=false
 export SKIP_TAGS="rpmcopy,subscribe"
 export PACKER_ONLY_EXCEPT=--only=amazon-ebs.fedora-38-x86_64,amazon-ebs.fedora-38-aarch64
 
+# wait until the rpms are built upstream
+COMMIT_SHA="${COMMIT_SHA:-$(git rev-parse HEAD)}"
+while true; do
```

Comment:
Can we do this with a for loop so we wait for example for a max of 10 iterations or 5 minutes? (or another sensical value).

Reply:
considering upstream rpm builds might take an hour or sometimes more, I'd prefer to just use the jenkins timeout.